schema.json

```diff
@@ -93,24 +99,24 @@
       },
       {
          "name": "frequence_calcul",
-         "description": "Fréquence de calcul, de collecte ou de parution de l’indicateur.",
-         "example": "Quotidienne",
+         "description": "Fréquence de calcul ou de collecte de l’indicateur. NB : Peut-être différente de la fréquence de publication : il est possible de monitorer un indicateur quotidiennement mais de ne le publier que tous les mois par exemple.",
+         "example": "quotidienne",
          "type": "string",
          "constraints": {
             "required": true,
             "enum": [
-               "Quotidienne",
-               "Hebdomadaire",
-               "Mensuelle",
-               "Annuelle",
-               "Trimestrielle",
-               "Continue"
+               "quotidienne",
+               "hebdomadaire",
+               "mensuelle",
+               "annuelle",
+               "trimestrielle",
+               "autre"
```

Comment:
There could be a large variety of frequencies (and the lack of clarity of `continue`), having an `autre` option is actually a good idea.
A user can't precise "autre : tous les deux mois" though? It would go in `commentaires` then?

Reply:
I'd say there are a couple of options here:
- leave it like this and let users specify in the comments or the indicator's name
- add more options to the enum (but I guess we'll never have all cases perfectly)
- add a second column like "other_frequency_detail" (optional) where alternative frequencies should be specified
Thought?